main.nf

```diff
@@ -220,6 +233,17 @@ params.bamutils_clip_left = 1
 params.bamutils_clip_right = 1 
 params.bamutils_softclip = false 
 
+//Pop Gen settings
+
+angsd_region_grch37 = "X:2699520-154931044"
```

Comment:
this should be a `params.angsd_region = ...` setting, ideally set to some defaults in the  `nextflow.config`.